### DISCUSSION TITLE
Why every 2 or 3 weeks once ChromeDriver having the update

### DISCUSSION THREAD

**Discussion by damodharheadrun** [ultrafunkamsterdam/undetected-chromedriver#1266] [Q&A] [answered]
May i know the reason behind we are upgrading the chromedriver version is this becase of the Google-chrome?

**Comment by disposablecarcass** [comment]
Yes, when chrome updates the chromedriver.exe must update.

**Reply by damodharheadrun** [reply, marked as answer]
Thank you for your information
